### ISSUE TITLE
Implement nodetool gettraceprobability

### ISSUE THREAD

**Issue by gnumoreno** [scylladb/scylla#7265]
Currently we don't have it:
```
nodetool gettraceprobability
nodetool: Found unexpected parameters: [gettraceprobability]
See 'nodetool help' or 'nodetool help <command>'.
```

`https://docs.datastax.com/en/cassandra-oss/3.0/cassandra/tools/toolsGetTraceProbability.html`

```
nodetool gettraceprobability
0
```

As a workaround we can use for now the API: 
`curl -X GET http://localhost:10000/storage_service/trace_probability`

**Comment by dyasny** [comment]
I think this is a bug - we should support all the c* 3.0 options, and the fact that the API has the option supports this

**Comment by tarzanek** [comment]
I've opened a PR, the default output however differs a bit from what you pasted @gnumoreno :
```
Current trace probability: 0.0
```
what output is current? resp. what output are we compatible against?

**Comment by tarzanek** [comment]
fwiw this was merged to java-tools as https://github.com/scylladb/scylla-tools-java/commit/4313155ab636209a773f05593c37eb5cae834a09

**Comment by slivne** [comment]
@roydahan  fix is being merged - need to write a test

**Comment by avikivity** [comment]
Fixed by d45df0c705aea1d1dca9d06021bee9fac5dcc8e7
